app/src/shared/components/TextField/NumberField/index.jsx

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import React, { useCallback, useState } from 'react'
+import cx from 'classnames'
+
+import SvgIcon from '$shared/components/SvgIcon'
+
+import styles from '../textField.pcss'
+
+type Props = {
+    className?: string,
+    type?: string,
+    min?: number,
+    max?: number,
+    step?: number,
+    hideButtons?: boolean,
+}
+
+const NumberField = ({ className, hideButtons, ...props }: Props) => {
+    const [value, setValue] = useState(null)
+
+    const addValue = useCallback((step) => {
+        let valueInt = Number.parseFloat(value)
```

Comment:
Thise `*Int` names. Where is the `Int` coming from? Does it indicated integer from the "old" days?